desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
+        .query({ visible_to_public: 'true', term: value })
+        .set('X-ACCESS-TOKEN', sd.CURRENT_USER.accessToken)
+        .end((err, res) => {
+          if (!err || res.ok) {
+            dispatch(updateArtistSuggestions(res.body))
+            dispatch(hideNotConsigningMessage())
+          }
+        })
+    } catch (error) {
+      console.error('error!', error)
```

Comment:
This is fine if you don't want to surface the error to the user in this case. For Redux you would typically `dispatch` an action that renders the error somewhere (like we [do in prediction](https://github.com/artsy/prediction/blob/master/src/actions/appActions.js#L30)).

Now that I look at this closer I realized there might be a more elegant way to handle this—

I see you're using an async function here which would allow you to await superagent, e.g.

````javascript
async (dispatch, getState) => {
  try {
    const res = await request.get(...)
    dispatch()
    dispatch()
  } catch (err) {
    console.error()
  }
}
````

Allowing you to let `await`ed errors throw without having to handle them explicitly in the callback `.end(err, res) => if !err`. You might not even need the `try`/`catch` if there's Redux thunk middleware that is meant to handle failed promises (can't tell if you're using that here).